theme_artists/views/snippets/s_company_team_spotlight.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_spotlight" inherit_id="website.s_company_team_spotlight" name="Artists s_company_team_spotlight">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc5" remove="o_cc1" separator=" "/>
+    </xpath>
+    <!-- Team Member 1 -->
+    <xpath expr="//div[hasclass('card')]" position="attributes">
+        <attribute name="class" add="o_cc4" remove="o_cc1" separator=" "/>
+    </xpath>
```

Comment:
It would better fit the theme styling without the roundness and without the border
 
Left: Now | Right: Suggestion
![image](https://github.com/user-attachments/assets/ffd5a101-afc2-4597-aad4-ae5f65231a9d)

```suggestion
    <!-- Team Member 1 -->
    <xpath expr="//div[hasclass('card')]" position="attributes">
        <attribute name="class" add="o_cc4 border rounded" remove="o_cc1" separator=" "/>
        <attribute name="style">border-width: 0px !important; border-radius: 0px !important;</attribute>
    </xpath>
```
The code suggestion above is based on what the web_editor gives when you enter 0 in roundness and border, but I find it weird that it adds the border and rounded class 🤷